core/src/main/scala/org/apache/spark/status/api/v1/AllStagesResource.scala

```diff
@@ -142,8 +142,10 @@ private[v1] object AllStagesResource {
       index = uiData.taskInfo.index,
       attempt = uiData.taskInfo.attemptNumber,
       launchTime = new Date(uiData.taskInfo.launchTime),
+      duration = uiData.taskInfo.duration,
```

Comment:
Could you create a method into TaskInfo for this line: https://github.com/apache/spark/blob/dbf842b7a8479f9566146192ffc04421591742d5/core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala#L890
 and use it here so that these two places have the same logic?